docs/Deployment/realtime-analytics.md

```diff
@@ -0,0 +1,88 @@
+# Analytics
+
+## Introduction
+
+[DIAL Core](https://github.com/epam/ai-dial-core) collects **system logs** and **chat completion logs**:
```

Comment:
@alexey-ban is this enough to differentiate system logs from chat completion logs?